docs/source/llms/chat-model-guide/index.rst

```diff
@@ -0,0 +1,802 @@
+Tutorial: Custom GenAI Models using ChatModel
+=============================================
+
+The rapidly evolving landscape of Generative Artificial Intelligence (GenAI) presents exciting opportunities and integration challenges.
+To leverage the latest GenAI advancements effectively, developers need a framework that balances flexibility with standardization. 
+MLflow addresses this need with the :py:class:`mlflow.pyfunc.ChatModel` class introduced in 
+`version 2.11.0 <https://mlflow.org/releases/2.11.0#chatmodel-interface-for-a-unified-chat-experience-with-pyfunc-models>`_, providing a 
+consistent interface for GenAI applications while simplifying deployment and testing.
+
+Purpose of this tutorial
```

Comment:
Could we also explain when users should use ChatModel at the top of this tutorial?
e.g. The model need to follow OpenAI input output format.

Reply:
Ah, I see this is included in the "Standardization" tab below. Given that it's not immediately visible (the user has to scroll and click through a couple tabs), we could probably still add some bullet points or brief paragraph here to make it clear when ChatModel is appropriate.

Reply:
Added a section at the top that compares and contrasts the two to answer the "why am I reading this?" question :)